robot/client/client.go

```diff
@@ -3,6 +3,7 @@ package client
 
 import (
 	"context"
+	"errors"
```

Comment:
one benefit of using stdlib `errors` over `github/pkg/errors` is that errors created using the former's `.New` function passes `test.ShouldResemble` tests but the latter does not

Reply:
> I'm making this change ahead of schedule here to help with testability: errors created with errors.New can be compared with test.ShouldResemble tests but github/pkg/errors cannot.

Thanks for sharing that! I did not know about the `test.ShouldResemble` compatibility.

I also have a gripe/motivation for preferring builtin `errors`: `github/pkg/errors.New` pulls in a stack trace that gets logged. That stacktrace is usually uninteresting and a waste of space.